ApiCheck/Comparer/ParameterComparer.cs

```diff
@@ -23,12 +23,28 @@ public override IComparerResult Compare()
 
     private void CompareDefaultValue()
     {
-      if (!Equals(ReferenceType.RawDefaultValue, NewType.RawDefaultValue))
+      if (!Equals(GetDefaultValue(ReferenceType), GetDefaultValue(NewType)))
       {
         ComparerResult.AddChangedProperty("Default Value", (ReferenceType.RawDefaultValue ?? "null").ToString(), (NewType.RawDefaultValue ?? "null").ToString(), Severities.DefaultValueChanged);
       }
     }
 
+    private object GetDefaultValue(ParameterInfo parameter)
+    {
+
+      if (parameter.ParameterType == typeof(DateTime) && (parameter.Attributes & ParameterAttributes.HasDefault) != ParameterAttributes.None)
```

Comment:
Would `Enum.HasFlag` make this a little clearer?

See: https://docs.microsoft.com/en-us/dotnet/api/system.enum.hasflag?view=net-6.0

Reply:
Might be a bit clearer, but comes at a cost: Enum.HasFlag requires boxing, see e.g. https://stackoverflow.com/questions/11665279/why-enums-hasflag-method-need-boxing

Reply:
Although you are right, this project was build as testing tool where performance is not the top priority.

Reply:
Good question. We'll be using it in a test environment as well, and thus not as reliant on performance or runtime, but I don't know of other uses. In general, I'm more a proponent of readable code, so I might incorporate the Enum.HasFlag suggestion. How do we want to decide?

Reply:
Please use `Enum.HasFlag`.

Reply:
@PMudra: I updated the the requested changes, to use Enum.HasFlag.